tsconfig.build.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "target": "ES2015",
+        "module": "ES2015",
+        "composite": true,
+        "moduleResolution": "node",
+        "noImplicitAny": true,
+        "noImplicitOverride": true,
+        "noUnusedParameters": true,
+        "noUnusedLocals": true,
+        "strictNullChecks": true,
+        "strict": true
+    },
+    "references": [
+        { "path": "packages/matter.js" },
+        { "path": "packages/node-matter" }
```

Comment:
packages/node ?

Reply:
This is the naming discussion we need to do and ideally the directly is names as the package in the end ... The question is how the "new" node-matter package is named ...

Reply:
@turon @mfucci Any ideas? I think Marco had @project-hip/matter-node.js in this PR, we could also leave node-matter ... This is then for the "final" PR

Reply:
Ideally the directories are named like the packages that are inside the directories ... So How the naming should be?

* @project-chip/matter.js we already have

Options for the node package in my eyes
* @project-chip/matter.js-node
* @project-chip/matter-node.js
* @project-chip/matter-nodejs
* ...

Reply:
Ok we decided for @project-chip/matter-node.js